packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -198,22 +198,22 @@
       color: #fff;
 
       &:hover {
-        background-color: darken($datepicker-selected-color, 5%);
+        background-color: $datepicker-selected-color;
       }
     }
 
     &--keyboard-selected {
-      background-color: lighten($datepicker-selected-color, 10%);
+      background-color: $datepicker-selected-color;
       border-radius: $datepicker-border-radius;
       color: #fff;
 
       &:hover {
-        background-color: darken($datepicker-selected-color, 5%);
+        background-color: $datepicker-selected-color;
```

Comment:
The background-color for hover should not be the same as `$datepicker-selected-color`. It should be `#0065a3` to match [hover-background-color for the emphasis button](https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/_variables.scss#L36). A new variable may be needed for this.

Reply:
Changed in 7802d23